components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -1,15 +1,50 @@
 <template>
   <div class="card mb-3 mt-5">
-    <div class="card-content">
+    <div class="card-content ">
       <div class="columns">
         <b-field class="column is-6 mb-0">
-          <b-input
+
+          <b-autocomplete
+            v-model="name"
+            :data = result
             placeholder="Search..."
-            type="search"
-            v-model="searchQuery"
             icon="search"
-            expanded>
-          </b-input>
+            open-on-focus
+            clearable
+            max-height="350px"
+            @keydown.native.enter="searchResult"
+            @keydown.native.up="moveUp"
+            @keydown.native.down="moveDown"
+            @typing="updateSuggestion"
+            @select="updateSelected"
+            >
+            <template slot-scope="props">
+
+                <div v-if="props.option.type==='History'">
+                  <div class="history"> {{props.option.name}} </div>
+                </div>
+
+                 <div v-else>
+                   <nuxt-link
+                    :to="{ name: 'rmrk-detail-id', params: { id: props.option.id }}"
+                    tag="div"
+                  >
+                  <div class="media">
+                  <div class="media-left">
+                    <b-image class="image is-32x32"
+                      :src="props.option.image === '' ? props.option.animation_url : props.option.image"
+                    >
+                    </b-image>
+                  </div>
+                  <div class="media-content">
+                      {{ props.option.name }}
+                  </div>
+                  </div>
+                  </nuxt-link>
+                </div>
+
+            </template>
```

Comment:
- identation
- spacing

Reply:
Btw @roiLeo, could you please post the eslint configurations that should be used? I used [this](https://www.digitalocean.com/community/tutorials/linting-and-formatting-with-eslint-in-vs-code) tutorial to setup eslint in vscode.

Reply:
> Btw @roiLeo, could you please post the eslint configurations that should be used? I used [this](https://www.digitalocean.com/community/tutorials/linting-and-formatting-with-eslint-in-vs-code) tutorial to setup eslint in vscode.

You don't have to setup anything, eslint rules are in `.eslintrc.js` file.
Next you'll need to run linter with command:
```bash
yarn lint
```
this will show warning and errors.

You can run with --fix flag to auto fix code:
```bash
yarn lint --fix
```

Reply:
Oh thanks. I guess you should add this to the `CONTRIBUTING.md`

Reply:
@pandasamanvaya good head's up! I've added it now https://github.com/kodadot/nft-gallery/commit/8c28268eed38139ea207b68bf23190eee17d57a1